lib/vendor/lime/lime.php

```diff
@@ -1186,8 +1199,7 @@ public function process($files)
 EOF;
       file_put_contents($tmp_file, $tmp);
       ob_start();
-      // see http://trac.symfony-project.org/ticket/5437 for the explanation on the weird "cd" thing
```

Comment:
**[PATCH] lime.php not working on windows when there are spaces in php interpreter path**

lime.php, line 541
```php
passthru(sprintf('%s "%s" 2>&1', $this->php_cli, $file), $return);
```

the command fails if the path to php interpreter contains spaces, this obviously prevent all test to run.

In order to work with executables in directories with spaces it should be
```php
passthru(sprintf('"%s" "%s" 2>&1', $this->php_cli, $file), $return);
```
but this fails too on windows: system/exec/passthru all fail when more than two arguments are quoted (it seems a php bug but I couldn't find any reference apart from some comments on the [system function page](https://www.php.net/manual/en/function.system.php))

In order to work on windows it should be written as
```php
passthru(sprintf('""%s" "%s"" 2>&1', $this->php_cli, $file), $return);
```
this works on windows but still fails on linux (note the double "double quotes": I need to quote the entire command not just the executable and the argument)

The only workaround I could find was something I found on the comments of the [system function](https://www.php.net/manual/en/function.system.php) that is adding a "nop" command call before the quoted command
```php
passthru(sprintf('cd & "%s" "%s" 2>&1', $this->php_cli, $file), $return);
```